submit-web/src/components/UserManagement/NewUser/NewUserForm.tsx

```diff
@@ -134,6 +155,21 @@ export default function NewUserForm() {
               <ControlledRadioGroup name="applicationAccess">
                 <FormOptions error={Boolean(errors["applicationAccess"])} />
               </ControlledRadioGroup>
+
+              <When condition={selectedRole === "CollaboratorSpecific"}>
+                <ControlledMultiSelect
+                  multiple
+                  selectAll
+                  name="project_ids"
+                  options={
```

Comment:
you can extract this out into a const and use useMemo so it doesn't recalculate on every render